invoke-ai/InvokeAI
# fix(ui): jank in depthanything model size select

## Summary

The select had a default search value, which meant it only showed "small" as an option on first load.

## Related Issues / Discussions

n/a

## QA Instructions

- Add a CA layer
- Expand advanced
- Set processor to depth anything
- Click the model size dropdown, it should show all 3 sizes

## Merge Plan

n/a

## Checklist
